### DISCUSSION TITLE
hi ! how to  adjust the  threshold

### DISCUSSION THREAD

**Discussion by JJun-Guo** [snakers4/silero-vad#228] [Q&A] [answered]
## ❓ Questions and Help

We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.

**Comment by snakers4** [comment, marked as answer]
https://github.com/snakers4/silero-vad/blob/7c671a75c2f82830a550ab4a182e86d3a2609d70/utils_vad.py#L119-L128
